### PR TITLE
TagManager

### PR DESCRIPTION
**Pull request by Nowp**

### CODE REVIEW

File: src/main/java/fr/ppptours/cafeteria/model/tags/TagManager.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * This class should be used whenever an entity can have an undefined number of a specific type of tag.
+ * @param <T> The type of tag we need to use.
+ */
+public class TagManager<T extends Tag> {
+
+    private final Set<T> tags;
```
Comment (Banalian) [inline src/main/java/fr/ppptours/cafeteria/model/tags/TagManager.java:14]: This variable might be missing some JavaDoc

**Review by Schizopinguin — APPROVED** [review]
Seems good to me, you can build your great tag

**Review by Banalian — APPROVED** [review]
Seems correct to me

**Review by jumsee — APPROVED** [review]
It's gooood

**Pull request merged**